Kitodo-FileManagement/src/test/java/org/kitodo/filemanagement/FileManagementTest.java

```diff
@@ -191,6 +191,25 @@ public void shouldDeleteDirectory() throws IOException {
     }
 
     @Test
+    public void shouldGetSizeOfDirectory() throws Exception {
+        int testContent = 156575;
+
+        URI resource = fileManagement.create(URI.create("fileTest"), "size.txt", true);
+        Assert.assertTrue(fileManagement.fileExist(resource));
+
+        OutputStream outputStream = fileManagement.write(URI.create("fileTest/size.txt"));
+        try {
+            outputStream.write(testContent);
+        } finally {
+            outputStream.flush();
+            outputStream.close();
+        }
+
+        long directorySize = fileManagement.getSizeOfDirectory(URI.create("fileTest"));
+        Assert.assertEquals(directorySize, 1);
```

Comment:
Content of tested directory contains 3 and not 1 entry. Is the test scenario correct?

Reply:
I'm not sure what is wrong. On my computer size of directory is 1...